api/v1beta1/hostedcluster_types.go

```diff
@@ -610,12 +610,22 @@ type CIDRBlock string
 
 // APIServerNetworking specifies how the APIServer is exposed inside a cluster
 // node.
+// +kubebuilder:validation:XValidation:rule="(!has(self.advertiseAddress) && !has(self.advertiseAddresses)) || (has(self.advertiseAddress) && !has(self.advertiseAddresses)) || (!has(self.advertiseAddress) && (has(self.advertiseAddresses) && self.advertiseAddresses.size() > 0))", message="AdvertiseAddress and AdvertiseAddresses are exclusive between themselves"
 type APIServerNetworking struct {
 	// AdvertiseAddress is the address that nodes will use to talk to the API
 	// server. This is an address associated with the loopback adapter of each
-	// node. If not specified, 172.20.0.1 is used.
+	// node. If not specified, the controller will look for the first IP on each
+	// CIDR (IPv4 and IPv6). If the controller cannot infer the first IP of the CIDR,
+	// the default values will be set 172.20.0.1 or/and fd00::1.
```

Comment:
Please update this comment to say that if we don't specify the IP, then the default value will be used (not the first of the CIDR)